### PR TITLE
DDF-3643 exclude colors from search indicative of errors

### PR DESCRIPTION
**Pull request by GabrielFabian**
#### What does this PR do?
The random color generator for searches uses varying degrees of Red, Orange, Yellow.This PR will omit colors like these that usually indicate to the user that there is an error or warning. 

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@lcrosenbu @bdeining

#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@bdeining
@jlcsmith 

#### What are the relevant tickets?
[DDF-3643](https://codice.atlassian.net/browse/DDF-3643)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/ui/catalog-ui-search/src/main/webapp/js/ColorGenerator.js [hunk 1]
```diff
@@ -30,12 +30,20 @@ define([
                 '#6a3d9a',
                 '#fdbf6f',
                 '#ff7f00',
-                '#fb9a99',
-                '#e31a1c',
                 '#b2df8a',
                 '#33a02c',
                 '#a6cee3',
-                '#1f78b4'
+                '#1f78b4',
```
Comment (jlcsmith) [inline catalog/ui/catalog-ui-search/src/main/webapp/js/ColorGenerator.js]: ❓ I could be wrong, but I think we were trying to ensure these colors were colorblind safe.  Did you check for this when adding the new colors?

Comment (GabrielFabian) [reply catalog/ui/catalog-ui-search/src/main/webapp/js/ColorGenerator.js]: The colors that were added came directly from @lcrosenbu , so I'm assuming so... But that's a great question. I'll find out.

**Comment by andrewkfiedler** [comment]
Only ten of these colors will get used at a time, there's not much point to having more at the moment.

**Comment by GabrielFabian** [comment]
@andrewkfiedler what if you have more than 10 searches? Won't you then have duplicate colors?

**Comment by rzwiefel** [comment]
@GabrielFabian Intrigue only allows a max of 10 searches per workspace

**Comment by GabrielFabian** [comment]
@rzwiefel I wasn't aware of that. Thanks for the clarification.

**Comment by bdeining** [comment]
build now

**Comment by bdeining** [comment]
@lcrosenbu any update on the use of color-blind safe colors?

**Comment by jlcsmith** [comment]
build now

**Comment by GabrielFabian** [comment]
build now

**Comment by jlcsmith** [comment]
build now

**Comment by jlcsmith** [comment]
@GabrielFabian - 2 of the colors look pretty similar.  
capture

**Comment by GabrielFabian** [comment]
@jlcsmith I would agree that those two colors are probably too similar. @andrewkfiedler @lcrosenbu any opinions/suggestions?

**Comment by coyotesqrl** [comment]
Just a reminder that we should be designing with color-blindness in mind as well. Similar colors are definitely bad, but even dissimilar colors may not be sufficient. Is there maybe a color blind-sensitive javascript library or CSS post-processor that might help? Something that can both help us pick colors and perhaps apply patterns where color is not enough?

**Comment by andrewkfiedler** [comment]
http://mkweb.bcgsc.ca/colorblind/ has a lot of good info for dealing with colors and colorblindness.

**Comment by andrewkfiedler** [comment]
Here's another that gives a palette suggestion: https://ux.stackexchange.com/questions/94696/color-palette-for-all-types-of-color-blindness

**Comment by andrewkfiedler** [comment]
I tested out the color palette found in the previous comment and played with it a bit to come up with the 10 best.  
See https://github.com/codice/ddf/pull/3039

**Pull request closed without merging**